python/ray/dag/dag_node_operation.py

```diff
@@ -13,27 +13,28 @@
 
 class _DAGNodeOperationType(Enum):
     """
+    [CL]
     There are three types of operations that a DAG node can perform:
     1. READ: Read from an input channel.
     2. COMPUTE: Execute the method corresponding to the node.
     3. WRITE: Write to an output channel.
     """
 
-    READ = "READ"
+    NCCL_READ = "NCCL_READ"
```

Comment:
We should only have COMPUTE nodes after this PR. Why do we still need NCCL_READ and NCCL_WRITE?

Reply:
It was WIP. Removed NCCL_READ and NCCL_WRITE.